src/features/filters/FiltersPanelFooter.jsx

```diff
@@ -141,6 +162,25 @@ const FiltersPanelFooter = () => {
           </TooltipContent>
         </Tooltip>
       )}
+      {hasRole(userRoles, WRITE_IMAGES_ROLES) && (
```

Comment:
I think we'll want to check for `DELETE_IMAGES_ROLE` instead of `WRITE_IMAGES_ROLE` here.